src/main/java/uk/gov/ons/ctp/response/action/service/ActionService.java

```diff
@@ -295,4 +297,20 @@ public Action updateAction(final Action action) {
     }
     return existingAction;
   }
+
+  public void rerunAction(final Action action) {
+    log.with("action_Id", action.getId()).debug("Entering rerunAction");
```

Comment:
I don't think we should have logs that mention `Entering ...` instead better to say what we're doing i.e. 'Rerunning action'